src/pages/K8s.vue

```diff
@@ -225,6 +249,19 @@ export default {
         }
       }
     },
+    onChangeEngine(desiredEngine) {
+      ipcRenderer.invoke('settings-write', { kubernetes: { containerEngine: desiredEngine } })
+        .then(() => {
```

Comment:
This is probably easier to read as an async function? (`await ipcRenderer.invoke…`)

Reply:
sure, although now we're inconsistent.